### PR TITLE
[exporter/coralogixexporter] Add support for logs

### PR DESCRIPTION
**Pull request by povilasv**
**Description:** <Describe what has changed.>

Add support for logs

**Link to tracking Issue:** https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/12601

**Testing:** <Describe what testing was performed and which tests were added.>

- Local testing
- Unit testing

**Documentation:** 

- Added component documentation

### CODE REVIEW

**Review by oded-dd — APPROVED** [review]

File: exporter/coralogixexporter/factory.go [hunk 1]
```diff
@@ -32,6 +32,7 @@ func NewFactory() component.ExporterFactory {
 		createDefaultConfig,
 		component.WithTracesExporterAndStabilityLevel(createTraceExporter, stability),
 		component.WithMetricsExporterAndStabilityLevel(createMetricsExporter, stability),
+		component.WithLogsExporterAndStabilityLevel(createLogsExporter, stability),
```
Comment (codeboten) [inline exporter/coralogixexporter/factory.go:35]: Should the stability level be different for logs as it is the initial implementation?

File: exporter/coralogixexporter/metrics_client.go [hunk 2]
```diff
@@ -79,7 +79,7 @@ func (e *exporter) start(_ context.Context, host component.Host) (err error) {
 	headers["Authorization"] = "Bearer " + e.config.PrivateKey
 	e.metadata = metadata.New(headers)
 	e.callOptions = []grpc.CallOption{
-		grpc.WaitForReady(e.config.GRPCClientSettings.WaitForReady),
+		grpc.WaitForReady(e.config.Metrics.WaitForReady),
```
Comment (codeboten) [inline exporter/coralogixexporter/metrics_client.go:82]: Does this change need to be made in the trace client as well?

File: exporter/coralogixexporter/factory.go (hunk 1 above)
Comment (povilasv) [reply exporter/coralogixexporter/factory.go]: Changed to alpha, thanks!

File: exporter/coralogixexporter/metrics_client.go (hunk 2 above)
Comment (povilasv) [reply exporter/coralogixexporter/metrics_client.go:82]: Nope, this was a minor bug only in metrics client, we have 3 different configs:
```
e.Config - grpc trace settings
e.Metrics - grpc metrics settings
e.Logs - grpc logs settings
```

so the tracing one is actually using correct option :)

**Review by oded-dd — APPROVED** [review]

File: exporter/coralogixexporter/README.md [hunk 3]
```diff
@@ -1,13 +1,12 @@
 # Coralogix Exporter
 
-| Status                   |                 |
-| ------------------------ |-----------------|
-| Stability                | [beta]          |
-| Supported pipeline types | traces, metrics |
-| Distributions            | [contrib]       |
+| Status                   |                        |
+| ------------------------ |----------------------- |
+| Stability                | [beta, alpha for logs] |
```
Comment (codeboten) [inline exporter/coralogixexporter/README.md:5]:
```suggestion
| Stability                | traces, metrics [beta] |
|                          | logs [alpha]           |
```

File: exporter/coralogixexporter/README.md [hunk 4]
```diff
@@ -80,4 +82,4 @@ We love to assist our customers, simply [book your implementation session](https
 and we will walk you through setting up this exporter, step by step.
 
 [beta]:https://github.com/open-telemetry/opentelemetry-collector#beta
```
Comment (codeboten) [inline exporter/coralogixexporter/README.md:84]:
```suggestion
[alpha]:https://github.com/open-telemetry/opentelemetry-collector#alpha
[beta]:https://github.com/open-telemetry/opentelemetry-collector#beta
```

File: exporter/coralogixexporter/metrics_client.go (hunk 2 above)
Comment (codeboten) [reply exporter/coralogixexporter/metrics_client.go:82]: ah I see... `e.Config` = trace settings was confusing me :)

**Review by codeboten — APPROVED** [review]

**Pull request merged**